src/azure-cli/azure/cli/command_modules/acr/task.py

```diff
@@ -778,6 +781,9 @@ def acr_task_run(cmd,  # pylint: disable=too-many-locals
         update_trigger_token = base64.b64encode(update_trigger_token.encode()).decode()
 
     task_id = get_task_id_from_task_name(cmd.cli_ctx, resource_group_name, registry_name, task_name)
+
+    context_path = prepare_source_location(context_path, client_registries, registry_name, resource_group_name)
```

Comment:
Might be overkill to do this but it won't hurt to add a test for running acr task run with local context. This is if the operation returns/creates an arm resource with the context field. If so, the test should check that the created run resource contains the local context path...